pkg/backend/nvme_path.go

```diff
@@ -57,6 +57,16 @@ func (s *Server) CreateNVMfPath(_ context.Context, in *pb.CreateNVMfPathRequest)
 	}
 
 	multipath := ""
+	if numberOfPaths, ok := s.Volumes.NvmeNumberOfPaths[in.NvMfPath.ControllerId.Value]; ok {
+		if numberOfPaths < 1 {
+			log.Printf("error: Entry with number of paths for %v exists with zero paths", in.NvMfPath.ControllerId.Value)
+			return nil, status.Error(codes.Internal, "Internal error")
+		}
+		// set multipath parameter only when at least one path already exists
+		multipath = strings.ToLower(
+			strings.ReplaceAll(controller.Multipath.String(), "NVME_MULTIPATH_", ""),
```

Comment:
usually you make the `replace` code a function, no ?

Reply:
only if it is used in more than one place

Reply:
but I don't mind extracting a func for this

Reply:
Updated